### ISSUE TITLE
Script fails with 'body' selector

### ISSUE THREAD

**Issue by eko24ive** [Artinov/home-work-8-Serhii7#2]
If I'll enter text "body" into **Selector** field and click "Set text" button
![image](https://cloud.githubusercontent.com/assets/25591824/23497353/53895fb0-ff2b-11e6-9b8a-25c1a1b5b5d7.png)

It won't do anything on the page and will show unhandled error in console:
![image](https://cloud.githubusercontent.com/assets/25591824/23497400/82e99f36-ff2b-11e6-9389-b27da96acce5.png)

This happens because you "auto prefix" given text with `.wrapper ` which is actually good point - it won't allow me to edit elements outside of `.wrapper` element, but there is no element by selector `body .wrapper` as you can see:
![image](https://cloud.githubusercontent.com/assets/25591824/23498195/1cd978e8-ff2f-11e6-9934-39527e251fb1.png)

So even your error handling doesn't help much:
```js
else {
  alert('Типа Фатал Эрор');
}
```

You can do following - either create if statement to check value of `str`:
```js
if(str == null) {
  alert("something went wrong");
  return; //to stop function execution 
}
```

Or wrap your code into `try..catch` statement which actually will handle any possible error:
```js
try {
if(str.length != 0){
    str.innerHTML = text;
  } else {
    alert('Типа Фатал Эрор');
  }
} catch (e) {
  alert("something bad happened: " + e.message)
}
```
You can read more about `try..catch` statement here https://www.w3schools.com/js/js_errors.asp

**Comment by Serhii7** [comment]
I`ve fixed, changed if(str ==true) and wrap my code into try...catch, because i have never used this statement.

**Comment by eko24ive** [comment]
Yeah, I see some improvements, but I'm getting an error when trying to `setText` for `p` selector:
![](http://i.imgur.com/fVEfiF8.gif)
